packages/player/src/styles/scss/components/cue-bar.scss

```diff
@@ -1,40 +1,58 @@
-.cueplayer-react .cueplayer-react-cue-bar {
+.cueplayer-react-cue-bar {
   display: flex;
   width: 100%;
-  height: 32px;
-
+  height: 24px;
+  position: relative;
   @include background-color-with-alpha(
-                  $cueplayer-react-primary-background-color,
-                  $cueplayer-react-primary-background-transparency
+    $cueplayer-react-primary-background-color,
+    $cueplayer-react-primary-background-transparency
   );
+  .cueplayer-react-cue-note {
+    position: absolute;
+    width: 6px;
+    cursor: pointer;
+    height: 100%;
+    background-color: white;
+  }
+  .cueplayer-react-cue-note-learner {
+    position: absolute;
+    height: 100%;
+    width: 6px;
+    background-color: orange;
+    cursor: pointer;
+  }
+  .cueplayer-react-cue-note-inactive {
+    opacity: 0.5;
+  }
+  .cueplayer-react-cue-note-active {
+    opacity: 1;
+  }
+  .cue-popup-content {
```

Comment:
for consistency we should consider the same prefix for these as all the other classes `cueplayer-react`